src/duqtools/submit.py

```diff
@@ -48,12 +48,12 @@ def job_scheduler(queue: Deque[Job], max_jobs=10):
     tasks: Deque[Job] = deque()
     completed: Deque[Job] = deque()
 
-    for _ in range(max_jobs):
-        job = queue.popleft()
-        task = job.start()
-        tasks.append(task)
+    while tasks or queue:
+        if queue and len(tasks) < max_jobs:
```

Comment:
I'm wondering if this should be a `while` so that we instantly fill the task deque.

```suggestion
        while queue and len(tasks) < max_jobs:
```

Reply:
I think submitting 1 job each second is much better for systems like the gateway. Jonathan had an edge case where submitting simultaneously caused the system to crash. Keeps the system from exploding

Reply:
Yeah, good point. I think this happens anyway, because there is a `time.sleep(interval)` in the parent loop, so only 1 task gets submitted every interval. This change just fills up the queue in one sweep (as it did before).